v2/spanner-to-sourcedb/src/test/resources/SpannerToSourceDbInterleaveMultiShardIT/spanner-schema.sql

```diff
@@ -29,7 +29,7 @@ CREATE TABLE IF NOT EXISTS child21 (
   in_ts TIMESTAMP,
   migration_shard_id STRING(50),
 )  PRIMARY KEY(id, child_id),
- INTERLEAVE IN PARENT parent2 ON DELETE CASCADE;
+ INTERLEAVE IN parent2;
```

Comment:
Same as earlier, should this schema be modified in place, or does there need to be another table added which contains `INTERLEAVE IN` while preserving the earlier `INTERLEAVE IN PARENT parent2 ON DELETE CASCADE` ?

Reply:
Added